docs/src/SBP_Operators.md

```diff
@@ -0,0 +1,182 @@
+- **Robustness**: Methods are less sensitive to parameter choices and grid resolution, leading to more reliable simulations
+
+SBP operators achieve this by providing:
+
+- **Provable stability**: Mathematical guarantees about the behavior of the numerical method through discrete energy estimates
+- **Exact conservation**: Discrete conservation laws that hold to machine precision
+- **Flexibility**: A unified framework that encompasses finite differences, finite elements, and spectral methods
+
+### Historical Context and Development
+
+The development of SBP operators represents a significant advancement in bridging the gap between the theoretical foundations of finite element methods and the computational efficiency of finite difference methods. While finite element methods naturally possess stability properties through their variational formulation, finite difference methods traditionally lacked such systematic guarantees. SBP operators provide the missing link by bringing the stability analysis tools of finite element methods to finite difference and other numerical approaches.
+
+The framework has since been extended to encompass a much broader class of methods, making it a unifying principle in numerical analysis. This generality allows practitioners to choose the most appropriate discretization for their specific problem while maintaining the same theoretical guarantees and structure-preserving properties.
+
+[^LampertRanocha2024]:
+    Lampert, Ranocha (2024):
+    Structure-Preserving Numerical Methods for Two Nonlinear Systems of Dispersive Wave Equations
+    [arXiv: 2402.16669](https://arxiv.org/abs/2402.16669)
+
+## 2. Mathematical Foundation
+
+### The Core SBP Property
+
+The fundamental property that defines a first-derivative SBP operator is:
+
+```math
+MD + D^T M = t_R t_R^T - t_L t_L^T
+```
+
+where:
+
+- ``D`` is the derivative operator (matrix)
+- ``M`` is the symmetric, positive definite mass matrix
+- ``t_L = (1,0,...,0)^T`` and ``t_R = (0,...,0,1)^T`` extract boundary values
+
+This property is the discrete analog of integration by parts:
+
+```math
+\begin{aligned}
+\underbrace{ \boldsymbol{u}^T M D \boldsymbol{v} + \boldsymbol{u}^T D^T M \boldsymbol{v} }_{\displaystyle \approx \int_{x_{\min}}^{x_{\max}} u\, (\partial_x v) + \int_{x_{\min}}^{x_{\max}} (\partial_x u)\, v } 
+&= 
+\underbrace{ \boldsymbol{u}^T \boldsymbol{t}_R \boldsymbol{t}_R^T \boldsymbol{v} - \boldsymbol{u}^T \boldsymbol{t}_L \boldsymbol{t}_L^T \boldsymbol{v} }_{\displaystyle \approx u(x_{\max})\,v(x_{\max}) - u(x_{\min})\,v(x_{\min}) }.
+\end{aligned}
+```
+
+where for periodic SBP operators their property naturally simplifies to
+
+```math
+MD + D^T M = 0.
+```
+
+### Understanding the Mass Matrix
+
+The mass matrix ``M`` approximates the continuous inner product:
+
+```math
+\langle u, v \rangle_M = u^T M v \approx \int_{x_{min}}^{x_{max}} u(x) v(x) dx
```

Comment:
```suggestion
\langle \boldsymbol{u}, \boldsymbol{v} \rangle_M = \boldsymbol{u}^T M \boldsymbol{v} \approx \int_{x_{min}}^{x_{max}} u(x) v(x) dx
```
On the left-hand side we are dealing with vectors, on the right-hand side with functions.